ocpkg

```diff
@@ -871,9 +871,8 @@ rm -rf link-grammar*
 if [ $# -eq 1 ]; then
   local _version=${1}
 else
-  local _version=$(git ls-remote -t https://github.com/opencog/link-grammar \
-    | grep -v "\^" | awk -F "-" '{print $3}' \
-    | sort -t "." -k1,1n -k2,2n -k3,3n | tail -n 1)
+  local _version=$(wget -O - http://www.abisource.com/downloads/link-grammar/current/ \
+    | grep md5 | awk 'match($0, /[5-9]\.[0-9]+\.[0-9]+/) {print substr($0, RSTART, RLENGTH)}')
 fi
 wget https://github.com/opencog/link-grammar/archive/link-grammar-${_version}.tar.gz
```

Comment:
This is insane.  Avoid all the complexity. Just get `http://www.abisource.com/downloads/link-grammar/current/link*.tag.gz` and be done with it.   Why are you fighting me on this?  This was set up to work correctly 8 or 9 years ago, why are you making yourself miserable, and making me miserable? Why not just do the simplest, easiest thing? Why add complexity, when that complexity literally makes things worse, less stable, buggier and more prone to failure?

Reply:
> This is insane. Avoid all the complexity. Just get `http://www.abisource.com/downloads/link-grammar/current/link*.tag.gz` and be done with it. 

I either have to do that get your signature and verify the download or just get the tarball through https from github. I choose the latter. I am getting the version details from abisource for now, because you broke the version tags on github.

> Why are you fighting me on this? 

Not fighting you.

>This was set up to work correctly 8 or 9 years ago, why are you making yourself miserable, and making me miserable?

Actually it was using github tag archive for 1 year without any issue. So, not making myself miserable and definitely not thinking of making you miserable, as you have stated multiple times you don't use this script.

Reply:
> you don't use this script.

I don't use this script, but I get emails from users who do use this script, and I end up doing support for them.  I've wasted weeks of my life dealing with emails and bug reports from users who try to build from github tarballs. Its not fun at all, and I will never get those weeks back. Time is precious. Wasting time is evil. Wasting one's life fighting with github stupidities is ... well, I get annoyed when I'm forced to waste time.  (I think there's a song by Rush called "Tom Sawyer" and it has a line about "the friction of the day.")

As you well know, I am easy to annoy, and I am not fun to be around when I'm annoyed.  I don't like being not-fun, it is annoying. I would rather be fun to be around all the time!